capstone_project/src/main/webapp/pages/analyze.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>STEP Pod 155 Capstone</title>
+    <link rel="stylesheet" href="../style.css">
+    <script src="../scripts/comments.js"></script>
```

Comment:
is this the correct script file?

Reply:
Ya, I was being dumb and put my stuff in the comments page initially. :( it is your script file, sorry!!